lib/deserializers.php

```diff
@@ -161,3 +161,31 @@ function elementList(Reader $reader, $namespace = null) {
     return $values;
 
 }
+
+/**
+ * @param Reader $reader
+ * @param $popo
+ * @param $namespace
+ * @return null|$popo
+ */
+function popoObject(Reader $reader, $popo, $namespace) {
+    if ($reader->isEmptyElement) {
+        $reader->next();
+        return null;
+    }
+
+    $reader->read();
+    do {
+
+        if ($reader->nodeType === Reader::ELEMENT && $reader->namespaceURI == $namespace) {
```

Comment:
Actually... I changed my mind!

I thought using `keyValue` this function would become super short, but that's not true... you would still need to loop through the result of `keyValue` and filter out the namespaced elements... which sucks.